docs/docs/docs/sbt/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+title: sbt plugin
+url: /docs/sbt
+---
+
+# What is `sbt-compendium`?
+
+`sbt-compendium` is a plugin that provides you client for
+[compendium](https://github.com/higherkindness/compendium). It allows
+you to communicate with a compendium service.
+
+# Getting started
+
+Add the following line to your `project/plugin.sbt`:
+
+```mdoc
+    addSbtPlugin("io.higherkindness" %% "sbt-compendium" % "0.0.1")
+```
+
+The following will also need to be added to your project settings in `build.sbt`:
+
+```mdoc
+    .settings(
+         compendiumSrcGenProtocolIdentifiers := List(ProtocolAndVersion("supplier",None),ProtocolAndVersion("material",None),ProtocolAndVersion("sale",None)),
+         compendiumSrcGenServerHost := "localhost",
+         compendiumSrcGenServerPort := 8080,
+         sourceGenerators in Compile += Def.task {
+           compendiumSrcGenClients.value
+         }.taskValue
+    )
+```
```

Comment:
```suggestion
```
.settings(
   compendiumSrcGenProtocolIdentifiers := List(ProtocolAndVersion("supplier",None),ProtocolAndVersion("material",None),ProtocolAndVersion("sale",None)),
   compendiumSrcGenServerHost := "localhost",
   compendiumSrcGenServerPort := 8080,
   sourceGenerators in Compile += Def.task {
     compendiumSrcGenClients.value
   }.taskValue
)
```
```